src/pymedphys/dicom/__init__.py

```diff
@@ -24,22 +24,48 @@
 # program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
 
 
-"""A Dicom toolbox built ontop of the pydicom library.
+"""A DICOM toolbox. Available functions include:
 
-Available Functions
--------------------
+General
```

Comment:
Basically was thinking it would be useful to expand the docs with explanations/examples, and I figured it might help to start by categorising our DICOM functions. Things blew up in complexity from there, so I thought I'd stop and see what your thughts were, @SimonBiggs, @kingrpaul and anyone else :)

Reply:
Go for it. Take the liberty to rework it :)